chorus_lib/src/core.rs

```diff
@@ -65,9 +63,21 @@ where
     }
 }
 
+/// If the value implements `Clone`, the located value of the same type also implements `Clone`.
+impl<V: Clone, L1> Clone for Located<V, L1>
```

Comment:
This is useful when we need to unwrap a located value (of clonable type) that we'll use later.